From cfdc7141a5e50dea042b8bd5fa8c070d5ce1e216 Mon Sep 17 00:00:00 2001
From: thekinhnguyen <the.nguyen.vx@renesas.com>
Date: Fri, 26 Feb 2021 17:29:20 +0700
Subject: [PATCH 065/135] arm64: dts: renesas: r9a07g044l: Add SPI Multi IO
 node

Add SPI Multi IO (SPIBSC) node for RZ/G2L (R9A07G044L) SoC.

Signed-off-by: thekinhnguyen <the.nguyen.vx@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g044l.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
index ace1710..3753f3a 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
+++ b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
@@ -312,6 +312,18 @@
 			eth-pin-mode-switch = <&pinctrl 1>;
 			status = "disabled";
 		};
+
+		spibsc: spi@10060000 {
+			compatible = "renesas,r9a07g044l-spibsc";
+			reg = <0 0x10060000 0 0x78>, <0 0x20000000 0 0x10000000>;
+			interrupts = <GIC_SPI 41 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&cpg CPG_MOD R9A07G044L_CLK_SPI>;
+			resets = <&cpg R9A07G044L_CLK_SPI>;
+			power-domains = <&cpg>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "disabled";
+		};
 	};
 
 	timer {
-- 
2.7.4

